python/lsst/daf/persistence/repository.py

```diff
@@ -356,44 +291,16 @@ def getKeys(self, *args, **kwargs):
         Get the keys available in the repository/repositories.
         :param args:
         :param kwargs:
-        :return: A dict of {key:valueType} or a list of these dicts, depending on the parentJoin rules.
-        """
-        return self.doParents(Repository.doGetKeys, *args, **kwargs)
-
-    def doGetKeys(self, *args, **kwargs):
-        """Get the keys from this repository only. Typically this function is called only by doParents, and
-        other classes should call getKeys.
-
-        :param args:
-        :param kwargs:
         :return: A dict of {key:valueType}
         """
         # todo: getKeys is not in the mapper API
         if self._mapper is None:
             return None
-        return self._mapper.getKeys(*args, **kwargs)
+        keys = self._mapper.getKeys(*args, **kwargs)
```

Comment:
assigning keys to a variable before returning it improves the ability to debug it. I needed it for something.